packages/account/src/Sections/Profile/LanguageSettings/language-settings.tsx

```diff
@@ -1,21 +1,31 @@
 import { Redirect } from 'react-router-dom';
-import { routes } from '@deriv/shared';
+import { BLOCK_LANGUAGES, routes } from '@deriv/shared';
 import { observer, useStore } from '@deriv/stores';
-import { localize, getAllowedLanguages } from '@deriv/translations';
+import { useTranslations, getAllowedLanguages } from '@deriv-com/translations';
 import FormSubHeader from '../../../Components/form-sub-header';
 import LanguageRadioButton from '../../../Components/language-settings';
+import { useEffect } from 'react';
 
 const LanguageSettings = observer(() => {
     const { client, common, ui } = useStore();
+    const { switchLanguage, currentLang, localize } = useTranslations();
     const { has_wallet } = client;
+    // [TODO]: Remove changeSelectedLanguage() when whole app starts to use @deriv-com/translations
     const { changeSelectedLanguage, current_language } = common;
+
     const { is_mobile } = ui;
```

Comment:
are we not supposed to use useDevice for isMobile here?

Reply:
I think that is handled in the tablet view PR